--- ./mineclone2/mods/ITEMS/mcl_barrels/init.lua	2023-09-04 19:20:48.000000000 -0300
+++ ./Build/mods/ITEMS/mcl_barrels/init.lua	2023-09-05 15:52:28.763211395 -0300
@@ -67,7 +67,7 @@
 
 	minetest.swap_node(pos, { name = "mcl_barrels:barrel_open", param2 = node.param2 })
 	open_barrels[playername] = pos
-	minetest.sound_play({name="mcl_barrels_default_barrel_open", gain=0.5}, {
+	minetest.sound_play({ name = "mcl_barrels_default_barrel_open", gain = 1 }, {
 		pos = pos,
 		max_hear_distance = 16,
 	}, true)
@@ -90,11 +90,10 @@
 	if not node then return end
 	if node.name == "mcl_barrels:barrel_open" then
 		minetest.swap_node(pos, { name = "mcl_barrels:barrel_closed", param2 = node.param2 })
-		minetest.sound_play({name="mcl_barrels_default_barrel_close", gain=0.5}, {
+		minetest.sound_play({ name = "mcl_barrels_default_barrel_close", gain = 1 }, {
 			pos = pos,
 			max_hear_distance = 16,
 		}, true)
-
 	end
 end
 
